interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: The parameter is a value placeholder that will be put into a function.  The argument is the actual input value to be used in the console.log
 
-Researched answer:
+Researched answer: Parameters act as placeholders for variables which will be invoked by the function.  After being placed in the parentheses, they can be utilized throughout the logic.  Arguments is the actual data passed through function invocation.  Arguments have to be a data type that Javascript understands.
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: The .map method takes the predefined parameters that are value, which is required, index and array, which are optional.
 
-Researched answer:
+Researched answer: The Higher Order Function .map method goes through, or iterates, the length of the array, or until conditions are met.  The three built-in, predefined parameters are: value, which is required, index, and array, in that order.  They can be named anything, but the order will remain that same.
```

Comment:
This would be a great place to dive deeper into the concepts. What can index and array be used for if we decide to use them as parameters?